### PR TITLE
updated portfolio endpoint

### PR DESCRIPTION
**Pull request by X3NOSIZ**

### CODE REVIEW

**Comment by phenax** [comment]
@X3NOSIZ, I'm not sure you can configure `.js.org` to work with this subdomain. You can use github pages.

**Comment by X3NOSIZ** [comment]
Umm like I can't use gh pages for some personal reasons, but I guess if this works on `.js.org` then it should also work here.

**Comment by phenax** [comment]
@X3NOSIZ, The problem is I'm not sure how you will configure the ssl certificate to include this subdomain. Let me know if you still want to try it out. I'll merge it in.

**Comment by X3NOSIZ** [comment]
@phenax I can do one thing which might fix this issue. I can deploy my site to Vercel & this might fix the issue. I'm pasting the link over here, can you let me know if it is working or not please?

**Comment by X3NOSIZ** [comment]
Here you go -> https://neilblaz3.vercel.app/

**Comment by phenax** [comment]
Oooof. Bad news man. Vercel has restrictions for subdomains (#249). I think for now, you should make it a URL redirection instead of a CNAME record. You can change it later if you want.

```js
  "URL": "https://neilblaz3.vercel.app",
```

**Comment by X3NOSIZ** [comment]
Let's do onething, I'm creating my new blog site, which will be ready by Nov. So let's keep `pratyay.is-a.dev` on hold. I'll be deploying the site on gh-pages soon & I'll let you know then ok?

**Comment by X3NOSIZ** [comment]
Here you go -> https://neilblaze.github.io/Neoblog/

File: domains/pratyay.json [hunk 1]
```diff
@@ -6,6 +6,6 @@
     "email": "neilblaze007@gmail.com"
   },
   "record": {
-    "CNAME": "neilblaze-v2.netlify.app"
+    "CNAME": "https://neilblaze.github.io/Neoblog"
```
Comment (phenax) [inline domains/pratyay.json:9]:
```suggestion
    "CNAME": "neilblaze.github.io"
```

**Review by phenax — CHANGES_REQUESTED** [review]

**Review by phenax — APPROVED** [review]

**Pull request merged**